BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01822-01
+2.0.0-prerelease-01901-01
```

Comment:
Given that we are probably going to take your change into release/uwp, I want to ask why did you require a buildtools update? I ask because that might bring unintended changes to release/uwp.